app/assets/stylesheets/govuk_publishing_components/components/_step-by-step-nav-related.scss

```diff
@@ -44,3 +44,27 @@
 .gem-c-step-nav-related__link-item {
   margin-top: govuk-spacing(3);
 }
+
+// stylelint-disable declaration-no-important
+@include govuk-media-query($media-type: print) {
+  .gem-c-step-nav-related {
+    background: unset;
+    border-color: $govuk-print-text-colour !important;
```

Comment:
Not sure the `important` is necessary here?

Reply:
@andysellick The `!important` is now removed.